ci_tests/published_codes_compilation_test.py

```diff
@@ -1,23 +1,26 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-from sys import exit
 import errno
 import json
 import os
 import socket
+import sys
 import xml.etree.ElementTree as ET
+from sys import exit
+
 import requests
 from ipolutils.read_text_file import read_commented_text_file
 
-import sys
-sys.path.append(os.path.join(os.path.dirname(__file__), "../ipol_demo/modules/"))
```

Comment:
Why remove this? it makes it so that the script can be called from anywhere in the filesystem